dev/ansible/01_create_intention.yml

```diff
@@ -0,0 +1,94 @@
+- name: set variable
+  set_fact:
+    consul_token: "master"
+
+#*************************************************************************************************#
+# mc => minio (allow)
+#*************************************************************************************************#
+- name: Check if intention mc => minio (allow) exists
+  shell: consul intention match -source mc
```

Comment:
Are you sure that you do not need a setup token?
Some of the CLI commands do not fail if the token is not provided, instead, it just returns an empty list. (when consul_acl_default_policy=deny)

It could be a case where this CLI command returns an empty list of intentions event though intentions exist

Reply:
I will make the changes to the ansible tasks, thanks :+1